GithubHandler.py

```diff
@@ -25,6 +25,21 @@ def __init__(
         self.prs = self.repo.get_pulls(state="open", sort="created", base=main_branch)
         self.prs_nums = [pr.number for pr in self.prs]
         self.prs_dict = {k: v for k, v in zip(self.prs_nums, self.prs)}
+        self.accepted_file_extensions = [
+            ".py",
+            ".java",
+            ".js",
+            ".ts",
+            ".html",
+            ".css",
+            ".scss",
+            ".sass",
+            ".gql",
+            ".graphql",
+            ".sql",
+            ".md",
```

Comment:
Adding a list of accepted file extensions is helpful for filtering in various methods. However, consider if it would be more beneficial to define this as a constant or configuration setting if the list might be subject to frequent updates or is to be reused.